### PR TITLE
Write Enumerable#grep_v specs

### PR DESCRIPTION
**Pull request by kachick**

### CODE REVIEW

File: core/enumerable/grep_v_spec.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      end
+      
+      it "raises an ArgumentError when not given a pattern" do
+        -> { @numerous.grep_v }.should raise_error(ArgumentError)
+      end
+    end
+    
+    describe "with block" do
+      it "returns an Array of matched elements that mapped by the block" do
+        @numerous.grep_v(@odd_matcher) { |n| n * 2 }.should == [0, 4, 8, 12, 16]
```
Comment (eregon) [inline core/enumerable/grep_v_spec.rb]: Interesting, I did not know grep_v (and grep) also have a "map" function in addition to the filtering/selection!

**Comment by eregon** [comment]
@kachick Seeing all your great commits recently,
I would like to propose you to be a RubySpec committer :tada:
So that you can push directly to master if you like :smiley: 
Would you like that?

**Comment by kachick** [comment]
@eregon Thank you! I'm honored :smile: 
I'll mainly use the commit rights for simple patch.

**Comment by elia** [comment]
:clap:

**Comment by eregon** [comment]
@kachick I sent you the invitation!

**Comment by kachick** [comment]
@eregon I received it. Thanks!

**Pull request merged**